content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,188 @@
+  
+1. Open the chat feature located on the left side bar. <br/><br/>
+    
+<img src = desktopChatButton.png
+    alt = "Chat Button (Screenshot)"/>
+
+2. Click the "Start new message" button and type in your user email. You should see your name and email pop up below, click it!<br/><br/>
+
+<img src = desktopStartNewMessage.png
+    alt = "start new message button (Screenshot)"/>
+<img src = desktopNameSelected.png
+    alt = "Name selected (Screenshot)"/>
+
+3. Send a friendly message to yourself to initiate the chat room. After sending the message, you should see a new chat room with yourself!<br/><br/>
+
+<img src = desktopJoinedChatRoom.png
+    alt = "Wallet + button (Screenshot)"/>
+
+4. Click the "Create a collaborative wallet" button to initiate the multisig configuration setup.<br/>
+<!-- Will need to update this image with new screenshot after fix -->
+<img src = desktopCreateCollabWalletBttn.png
+    alt = "collaborative wallet question (Screenshot)"/>
+
+5. Name your wallet and click "Continue".<br/><br/>
+    
+<img src = desktopWalletName.png
+    alt = "Create collaborative wallet view (Screenshot)"/>
+
+6. Add the number of keys you will be using in your wallet. Then, add the required keys to sign a transaction.<br/><br/>
+
+<img src = desktopWalletConfig.png
+    alt = "Configure collaborative wallet (Screenshot)">
+
+7. Be sure to check your configuration before clicking "Continue".<br/><br/>
+
+<img src = desktopCurrentConfig.png
+    alt = "your current configuration (Screenshot)">
+
+8. You can now review other properties of your wallet configuration, then continue.<br/><br/>
+
+<img src = desktopReviewWallet.png
+    alt = "Review collaborative wallet (Screenshot)">
+
+9. Assign the keys that are required for signing wallet transactions, and then continue.
+
+{{< notice tip >}}
+The key in green (Software) is an eligible key to sign with for your current device. Foreign keys are not eligible for signing locally and will need to be signed on the device they were imported or created on.
+{{< /notice >}}
+
+<img src = desktopAssignKeys.png
+    alt = "Assign key (Screenshot)">
+
+
+
+<!-- {{< notice note >}}
+You may be required to add the keys on the device they were imported locally to. If that is the case, proceed to step 10.
+{{< /notice >}} -->
+
+<!-- <img src = desktopErrorAddingKeys.png
+    alt = "adding key error (Screenshot)"> -->
+
+<!-- 10. Ensure your local key is selected (for me, that is my Desktop Key) and click "Continue".<br/><br/>
+
+<img src = desktopSelectKey.png
+    alt = "Assign Key Desktop (Screenshot)"> -->
+
+11. You should now see the original chat screen you created in step 2. This is where you can find the pending wallet configuration.<br/><br/>
+
+{{< notice tip >}}
+You will need to access this chat screen and configuration file to if you did not complete your setup.
+{{< /notice>}}
+
+<img src = desktopConfigDone.png
+    alt = "Chat screen pending keys (Screenshot)">
+
```

Comment:
I think we should add one more step: Finalize wallet. And clarify about the wallet finalization rule.

Typically for collaborative wallets, the rule is that the creator of the wallet has to be the one finalizing. However, for self-managed multisig, the wallet can be finalized from any device (since it's the same user).